nipype/pipeline/plugins/sge.py

```diff
@@ -147,7 +147,7 @@ def _qacct_verified_complete(taskid):
         while qacct_retries > 0:
             qacct_retries -= 1
             try:
-                strtaskid = '{0!d}'.format(taskid)
+                strtaskid = '{0:d}'.format(int(taskid))
                 proc = subprocess.Popen(
```

Comment:
i think i switched this to `CommandLine` in the slurm plugin - take a look there. `CommandLine` handles most encoding issues properly.